csrc/ir/nodes.cpp

```diff
@@ -4839,11 +4839,9 @@ std::vector<PolymorphicValue> SdpaBwdOp::evaluate(
   // Backward tensor inputs: grad_input, query, key, value, output, logsumexp,
   // cum_seq_q/k
   std::vector<at::Tensor> bwd_inputs;
-  for (auto idx : c10::irange(8)) {
+  for (auto idx : c10::irange(10)) {
```

Comment:
nitpick. The comment above should be updated
`grad_output`, ... , `max_q/k`

Reply:
Missed this comment, fixed this in the other PR.